roles/build-release/tasks/build.yaml

```diff
@@ -58,6 +58,12 @@
     chdir: "{{ playbook_dir | dirname }}"
     creates: "{{ _release_archive }}"
 
+# We can use this to test that the release on PyPi matches later (for example)
+- name: Record the sha256sum for the built tarball
+  command: sha256sum {{ _release_archive }}
+  changed_when: false
+  register: _tarball_checksum
```

Comment:
Shouldn't this be used somewhere?

Reply:
ompragash had a WIP to test the sha256 against what's published to pypi -- it'll be meant for that.
We can remove it until then if you feel strongly about it.

Reply:
No, just wanted to make sure that this wasn't an oversight :)